play/src/containers/GameInProgressContainer.tsx

```diff
@@ -26,6 +26,7 @@ export function GameInProgressContainer(props: GameInProgressContainerProps) {
   // retreives game data from react-router loader
   // if no game data, redirects to splashscreen
   const localModel = useLoaderData() as LocalModel;
+  const currentTimer = localModel.currentTimer;
```

Comment:
this will flag a linting error about object destructuring.

Reply:
Hm didn't seem to get an error here after linting. Do we not want to restructure the object here and just pass in localModel.currentTimer where we use currentTimer in the file instead?

Reply:
hmm, that's weird. I hope my linter settings are being carried over. I think for now we can just do `const { currentTimer} = localModel;` for consistency with the rest of the code